meinberlin/apps/kiezradar/serializers.py

```diff
@@ -96,10 +98,16 @@ def to_representation(self, instance):
             {"id": project_type.id, "name": project_type.get_participation_display()}
             for project_type in instance.project_types.all()
         ]
+        representation["status"] = [
+            {"id": project_status.id, "status": project_status.status}
+            for project_status in instance.status.all()
+        ]
 
         topics_enum = import_string(settings.A4_PROJECT_TOPICS)
         representation["topics"] = [
-            {"id": topic.id, "name": topics_enum(topic.code).label}
+            {"code": topic.code, "name": topics_enum(topic.code).label}
             for topic in instance.topics.all()
         ]
+
+        representation["query_text"] = instance.query.text if instance.query else ""
```

Comment:
I wasn't sure how to receive the `query_text` from the serializer, as `query` only resembles a primary key and no char